tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -205,3 +205,30 @@ describe('Test Self-Healing of resource modification when correctDrift option us
     })
   )
 });
+
+describe('Test resource behavior after deleting GitRepo using keepResources option for exisiting GitRepo', { tags: '@p1'}, () => {
+  qase(71,
+    it("Fleet-71: Test RESOURCES will be KEPT and NOT be DELETED after GitRepo is deleted when keepResources is set to true in existing GitRepo.", { tags: '@fleet-71' }, () => {
```

Comment:
Is it possible to blend this test under [describe('Test resource behavior after deleting GitRepo using keepResources option](https://github.com/rancher/fleet-e2e/blob/fd31f17ad53ef6319f8f421162d608c467ea3aaa/tests/cypress/e2e/unit_tests/p1_fleet.spec.ts#L104-L105) ? If not or complex not a blocker

Something like `if (keepResources === 'yes' && editConfig === true')`?

Reply:
I will give a try to it and update PR.

Reply:
After giving a try, I see it will require multiple `if-else` so will give a try afterwards